keycloak/keycloak-operator
# INTLY-6459 ensure client secret can be recreated

## JIRA ID
INTLY-6495

## Additional Information
The client secret is not recreated when deleted because the check only covers the client itself: if the client exists (but not the client secret), the operator will attempt to update the non-existing secret and fail with a nil pointer exception.

## Verification Steps
1. Deploy Keycloak
2. Create a client
3. Ensure the client secret is created
4. Delete the client secret
5. The operator should recreate the client secret

## Checklist:
